### PR TITLE
Add/ab test enhance gsuite post checkout notice

### PR DESCRIPTION
**Pull request by belcherj**
#### Changes proposed in this Pull Request

* Adds an AB Test to make the G Suite next steps notice more prominent. 

#### Testing instructions

* Purchase an extra G Suite license or G Suite
* Observe Original
* Ensure that you are in the test. Set variation (builderReferralBanner): In console run `localStorage.setItem( 'ABTests', '{"gSuitePostCheckoutNotice_20190211":"enhanced"}' );`
* Purchase an extra G Suite license or G Suite
* Observe variation

Original:
screen shot 2019-02-08 at 3 35 20 pm

Variation:
screen shot 2019-02-08 at 3 35 29 pm

### CODE REVIEW

**Comment by robertf4** [comment]
Testing description needs to be updated to use `localStorage.setItem( 'ABTests', '{"gSuitePostCheckoutNotice_20190208":"enhanced"}' )`.

**Comment by belcherj** [comment]
Updated just as you left the comment. :)

File: client/components/purchase-detail/index.jsx [hunk 1]
```diff
@@ -102,10 +103,15 @@ export default class PurchaseDetail extends PureComponent {
 			'is-placeholder': this.props.isPlaceholder,
 		} );
 
+		let requiredClass = 'purchase-detail__required-notice';
```
[the remark this answers is not in the document]
Comment (belcherj) [reply client/components/purchase-detail/index.jsx]: fixed

File: client/lib/abtest/active-tests.js [hunk 2]
```diff
@@ -138,4 +138,12 @@ export default {
 		},
 		defaultVariation: 'allPrimary',
 	},
+	gSuitePostCheckoutNotice: {
+		datestamp: '20190208',
```
[the remark this answers is not in the document]
Comment (belcherj) [reply client/lib/abtest/active-tests.js]: updated

**Pull request merged**